client/reader/recent/style.scss

```diff
@@ -56,9 +64,17 @@
 			}
 		}
 
+		.dataviews-view-list {
+			height: fit-content;
+			max-height: calc( 100vh - var( --masterbar-height ) - var( --content-padding-top ) - $feed-list-header-height - $feed-list-actions-height - $feed-list-footer-height - var( --content-padding-bottom ) );
+			overflow-y: auto;
+			@include custom-scrollbars-on-hover( transparent, $gray-600 );
```

Comment:
- Added custom scroll bar with same styling as our global scrollbar.
- Didn't updated the scrollbar styling of post content just to keep it consistent with other screens.